api/src/main/java/org/pmiops/workbench/genomics/GenomicExtractionService.java

```diff
@@ -274,4 +274,23 @@ public GenomicExtractionJob submitGenomicExtractionJob(DbWorkspace workspace, Db
 
     return new GenomicExtractionJob().status(TerraJobStatus.RUNNING);
   }
+
+  public void abortGenomicExtractionJob(String workspaceNamespace, String workspaceId, String jobId)
+      throws ApiException {
+    workspaceAuthService.getWorkspaceEnforceAccessLevelAndSetCdrVersion(
```

Comment:
nit: It looks like the convention for this service is to assume user access control is handled before these calls. This service is simply executing the request